apps/ideaspace/src/components/common/IdeaCard/IdeaCardImg.js

```diff
@@ -1,18 +1,22 @@
 import React from 'react';
 import Link from 'next/link';
+import Image from 'next/image';
+
 import { ImgButton } from './StyledIdeaCard';
 
-const IdeaCardImg = ({ cardId }) => {
+const IdeaCardImg = ({ cardId, ideaImage }) => {
   return (
     <Link href={`/ideaspace/workshop/${cardId}`}>
-      <ImgButton style={{ backgroundColor: '#FFFFFF' }}>
-        <img
+      <ImgButton style={{ backgroundColor: '#FFFFFF', width: '100%' }}>
+        <Image
           alt="idea_image"
-          src={`https://picsum.photos/seed/${cardId}/350/270?random=${cardId}`}
+          src={
+            ideaImage ||
+            `https://picsum.photos/seed/${cardId}/350/270?random=${cardId}`
+          }
+          width={353}
+          height={176}
```

Comment:
I know this was hardcoded previously, and out of the scope of this ticket, but thoughts on changing this to a relative value?